docs/labwc-actions.5.scd

```diff
@@ -124,9 +124,21 @@ Actions are used in menus and keyboard/mouse bindings.
 *<action name="Reconfigure" />*
 	Re-load configuration and theme files.
 
-*<action name="ShowMenu" menu="value" atCursor="yes" />*
+*<action name="ShowMenu"/>*
```

Comment:
This makes it seem as if `ShowMenu` as only argument would work. Suggest adding `menu="root-menu"` or something.

Reply:
Fixed.